client/js/libs/handlebars/parse.js

```diff
@@ -81,5 +81,8 @@ module.exports = function parse(text) {
 		}
 
 		return fragments;
+	}).join("") + linkParts.map((part) => {
+		const escapedLink = Handlebars.Utils.escapeExpression(part.link);
+		return `<div data-url="${escapedLink}"></div>`;
```

Comment:
Fair enough 😅 What do you recommend? Because right now rendering of something that needs escaping will fail.